bin/webpack-dev-server.js

```diff
@@ -317,8 +319,8 @@ function processOptions(wpOpt) {
 	// that wouldn't throw errors. E.g. both argv.port and options.port
 	// were specified, but since argv.port is 8080, options.port will be
 	// tried first instead.
-	options.port = argv.port === DEFAULT_PORT ? (options.port || argv.port) : (argv.port || options.port);
-	if(options.port) {
+	options.port = argv.port === DEFAULT_PORT ? defaultTo(options.port, argv.port) : defaultTo(argv.port, options.port);
+	if(options.port != null) {
```

Comment:
You should not add !=null here because it's not consistant with the current code style.
Should keep `if(options.port)`